Firestore/Source/CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ if(APPLE)
       API/FIRQuerySnapshot.mm
       API/FIRSnapshotMetadata+Internal.h
       API/FIRSnapshotMetadata.mm
+      API/FIRTimestamp.m
```

Comment:
This was not included, was it intentional?

Reply:
An omission.